discovery-provider/src/queries/health_check.py

```diff
@@ -92,3 +92,21 @@ def ipld_block_check():
 def ip_check():
     ip = helpers.get_ip(request)
     return success_response(ip, sign_response=False)
+
+@bp.route("/block_confirmation", methods=["GET"])
+def block_confirmation():
+    blockhash = request.args.get("blockhash")
+    blocknumber = request.args.get("blocknumber", type=int)
+    block_found, block_passed = False, False
+    bad_request = True
+    if blockhash is not None and blocknumber is not None:
+        bad_request = False
+        logger.info(f"ARGS: {blockhash, blocknumber}")
```

Comment:
maybe make this more obvious to grep for
logger.info(f"block_confirmation | ARGS: { }"{)

Reply:
lol this was actually forgotten to be removed. but yeah will do!